malasakit-django/pcari/templates/base_with_links.html

```diff
@@ -0,0 +1,5 @@
+{% extends 'base.html' %}
+{% blocktrans trimmed %}
+  This site uses the <a href="http://cafesystem.org/">Collective Assessment and Feedback Engine (CAFÉ)</a>.
+  Learn more about <a href="{{ about_url }}">CAFÉ</a>.
```

Comment:
This doesn't work because the `about_url` variable is defined by the `url` tag (here, `about_url` would be blank).